vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -155,10 +156,48 @@ void FlyoutShadowNode::createView() {
   m_flyoutClosedRevoker =
       m_flyout.Closed(winrt::auto_revoke, [=](auto &&, auto &&) {
         auto instance = wkinstance.lock();
-        if (!m_updating && instance != nullptr)
+        if (!m_updating && instance != nullptr) {
+          if (m_targetElement != nullptr) {
+            // When the flyout closes, attempt to move focus to
```

Comment:
Don't think there is anything wrong with this, but just calling out, that if the user doesn't set the target element of the flyout, we set it to the app window. I'm not sure where the focus will be in that case after the flyout dismisses.

Reply:
If there's a nested flyout that doesn't have a target element, then upon clicking the button to invoke that flyout, the outer flyout closes, but this happens in master to. I also tested the case where the parent flyout doesn't have a target. If I tab after the flyout dismisses, the dropdown at the top of the flyout example gets focused, so it seems that the app is able to get focus to a sensible place.